Tests/Foundation/Tests/TestListFormatter.swift

```diff
@@ -0,0 +1,108 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+#if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT
+    #if canImport(SwiftFoundation) && !DEPLOYMENT_RUNTIME_OBJC
+        @testable import SwiftFoundation
+    #else
+        @testable import Foundation
+    #endif
+#endif
+
+class TestListFormatter: XCTestCase {
+    private var formatter: ListFormatter!
+
+    override func setUp() {
+        super.setUp()
+
+        formatter = ListFormatter()
+    }
+
+    override func tearDown() {
+        formatter = nil
+
+        super.tearDown()
+    }
+
+    func test_locale() throws {
+        XCTAssertEqual(formatter.locale, Locale.autoupdatingCurrent)
+
+        formatter.locale = Locale(identifier: "en_US_POSIX")
```

Comment:
This is not portable (`en_US` is fine, `en_US_POSIX` is not a valid locale on Windows).